### PR TITLE
Create QOwnNotes Cask with v16.11.16

### PR DESCRIPTION
**Pull request by cornercase**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/qownnotes.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'qownnotes' do
+  version '16.11.16,b2545-185156'
+  sha256 '98e6546870a8b1202d6839ba0d1819f27b9b7f3ab626840e50135f106f8fe099'
+
+  # github.com/pbek/QOwnNotes was verified as official when first introduced to the cask
+  url "https://github.com/pbek/QOwnNotes/releases/download/macosx-#{version.after_comma}/QOwnNotes-#{version.after_comma}.dmg"
```
Comment (vitorgalvao) [inline Casks/qownnotes.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by cornercase** [comment]
Accepted all changes, but corrected second instance of version.after_comma to version.before_comma to create correct download URL.  Appcast included.

**Review by vitorgalvao — APPROVED** [review]

File: Casks/qownnotes.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'qownnotes' do
+  version '16.11.16,b2545-185156'
+  sha256 '98e6546870a8b1202d6839ba0d1819f27b9b7f3ab626840e50135f106f8fe099'
+
+  # github.com/pbek/QOwnNotes was verified as official when first introduced to the cask
+  url "https://github.com/pbek/QOwnNotes/releases/download/macosx-#{version.after_comma}/QOwnNotes-#{version.before_comma}.dmg"
+  appcast 'https://github.com/pbek/QOwnNotes/releases.atom',
+          checkpoint: '8428e3522fe51de301d61cf8e98fbe42b1ddab9a07e3c4ba03508086b683d6c6'
```
Comment (miccal) [inline Casks/qownnotes.rb]:
```
- appcast checkpoint mismatch
Expected: 8428e3522fe51de301d61cf8e98fbe42b1ddab9a07e3c4ba03508086b683d6c6
Actual: b50fae2bbdc57ad400b02198baec98eccef6731ee1343f6042129ffab3be441b
```

**Review by miccal — CHANGES_REQUESTED** [review]

**Comment by adidalal** [comment]
Merged and updated to latest version in https://github.com/caskroom/homebrew-cask/pull/27347


Check out `cask-repair`, it makes Cask updates like this quite easy (details in [CONTRIBUTING](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md))

ie. `cask-repair -v 16.12.0,b2568-113120 qownnotes`

**Pull request merged**